src/jobsautoreport/report.py

```diff
@@ -4,13 +4,30 @@
 
 from pydantic import BaseModel
 
-from jobsautoreport.models import JobState, JobType, StepState
+from jobsautoreport.models import JobState, JobType, MachineType, StepState
 from jobsautoreport.query import Querier
+from prowjobsscraper.equinix_usages import EquinixUsageEvent
 from prowjobsscraper.event import JobDetails
 
 logger = logging.getLogger(__name__)
 
 
+class MachineMetrics(BaseModel):
+    m3_small_x86: float = 0
```

Comment:
```
>>> m3_small_x86: float = 0
>>> type(m3_small_x86)
<class 'int'>
```
The type annotation in python is not evaluated by the interpreter, the value should be of explicitly of type float